frontend/components/CustomIcons/FavoritesIcon.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import getConfig from '../../helpers/getConfig';
+
+const { FavoritesSvg } = getConfig();
+
+/**
+ * The home icon component.
+ * @param {Object} props The icon component properties.
+ * @returns {JSX}
+ */
+const FavoritesIcon = props =>
+  (
+    <div {...props} dangerouslySetInnerHTML={{ __html: FavoritesSvg }} />
```

Comment:
I think we should avoid using dangerouslySetInnerHTML. Instead I think it would be better to to use the Icon component from the engage components library. 

``` import { Icon } from '@shopgate/engage/components'```

Have you tried this?

Reply:
Talked to @Carsten89 about that. We don't use the Icon component, because we want to be able to put the whole SVG in the config.

If we use the Icon component, we have to separate the SVG into content and viewBox.